pkgs/native_assets_cli/lib/native_assets_cli.dart

```diff
@@ -6,27 +6,26 @@
 /// build hook (`hook/build.dart`).
 library native_assets_cli;
 
-export 'src/api/architecture.dart' show Architecture;
 export 'src/api/asset.dart'
-    show
-        Asset,
-        DataAsset,
-        DynamicLoadingBundled,
-        DynamicLoadingSystem,
-        LinkMode,
-        LookupInExecutable,
-        LookupInProcess,
-        NativeCodeAsset,
-        StaticLinking;
+    show Asset, DataAsset, NativeCodeAsset, OSLibraryNamingExt;
 export 'src/api/build.dart' show build;
 export 'src/api/build_config.dart' show BuildConfig, CCompilerConfig;
-export 'src/api/build_mode.dart' show BuildMode;
 export 'src/api/build_output.dart' show BuildOutput, LinkOutput;
 export 'src/api/builder.dart' show Builder;
 export 'src/api/hook_config.dart' show HookConfig;
-export 'src/api/ios_sdk.dart' show IOSSdk;
 export 'src/api/link.dart' show link;
 export 'src/api/link_config.dart' show LinkConfig;
-export 'src/api/link_mode_preference.dart' show LinkModePreference;
 export 'src/api/linker.dart' show Linker;
-export 'src/api/os.dart' show OS;
+export 'src/architecture.dart' show Architecture, OSArchitecturesExt;
```

Comment:
Maybe don't export `OSArchitecturesExt` here, only in internal? it has no uses besides build_runner.

Reply:
Good call. I can move it to `package:native_assets_builder` package. I could move the test as well, but the test for it only tests the `OS.android.architectures` and is basically duplicating the list for android. So I'm not sure if it adds any value.